packages/commonwealth/client/scripts/controllers/chain/cosmos/gov/queries/useActiveCosmosProposalsQuery.ts

```diff
@@ -0,0 +1,23 @@
+import app from 'state';
+import { useQuery } from '@tanstack/react-query';
+import Cosmos from 'controllers/chain/cosmos/adapter';
+import { getActiveProposals } from '../utils';
+
+interface ActiveProposalsProps {
+  isApiReady?: boolean;
+}
+
+const useActiveCosmosProposalsQuery = ({
+  isApiReady,
+}: ActiveProposalsProps) => {
+  const cosmosChain = app.chain as Cosmos;
+  return useQuery({
+    /* eslint-disable @tanstack/query/exhaustive-deps*/
```

Comment:
ignoring here because `cosmosChain` is a whole object that I need for the request. 
React-query wants me to put the whole dependency into the array, but I just put the chain id. This still achieves what RQ wants without a super-long stringified key.

Reply:
Following convention and no need for comment anymore.